src/visibility.js

```diff
@@ -177,6 +191,45 @@ class Visibility extends BasePlugin {
     this._observer = null;
     this.eventManager.destroy();
   }
+
+  _dragMouseDown(e: MouseEvent) {
+    e.preventDefault();
+    // get the mouse cursor position at startup:
+    this._currMousePos.x = e.clientX;
+    this._currMousePos.y = e.clientY;
+    this.eventManager.listen(document, 'mouseup', this._dragStop.bind(this));
```

Comment:
maybe listen to this once and avoid needing to unregister in event handler

Reply:
Good idea